crates/bevy_pbr/src/material.rs

```diff
@@ -81,7 +92,13 @@ impl<M: Material> SpecializedMaterial for M {
     fn key(_material: &<Self as RenderAsset>::PreparedAsset) -> Self::Key {}
 
     #[inline]
-    fn specialize(_key: Self::Key, _descriptor: &mut RenderPipelineDescriptor) {}
+    fn specialize(
+        _descriptor: &mut RenderPipelineDescriptor,
+        _key: Self::Key,
+        _layout: &MeshVertexBufferLayout,
+    ) -> Result<(), SpecializedMeshPipelineError> {
```

Comment:
Why _ prefixes for variables that are used?

Reply:
Ah yeah it used to be an "empty" impl and I didn't update this. Nice catch!